src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityContractProperty.cs

```diff
@@ -128,13 +128,13 @@ internal sealed class GetAccessorSymbol : SourcePropertyAccessorSymbol
                        propertyModifiers,
                        location,
                        syntax,
-                       hasBody: false,
+                       hasBody: true,
                        hasExpressionBody: false,
                        isIterator: false,
                        modifiers: new SyntaxTokenList(),
                        MethodKind.PropertyGet,
                        usesInit: false,
-                       isAutoPropertyAccessor: true,
+                       isAutoPropertyAccessor: false,
```

Comment:
>isAutoPropertyAccessor [](http://example.com/codeflow?start=23&length=22)

It looks like we are going to stop synthesizing ```CompilerGeneratedAttribute``` for this method. Should be easy to mitigate, I think.

Reply:
@AlekseyTs I'm feeling the attribute should be on the whole property, not just the accessor. What do you think?